modules/dynamic-plugins/ref-example-third-party-plugin-installation.adoc

```diff
@@ -0,0 +1,77 @@
+[id="ref-example-third-party-plugin-installation_{context}"]
+= Example of installing a third-party plugin in {product}
+
+This section provides step-by-step instructions for integrating the link:https://github.com/backstage/community-plugins/tree/main/workspaces/todo/plugins[Todo plugin] into your {product-short}.
+
+. *Obtain the third-party plugin source code*: Clone the plugins repository and navigate to the link:https://github.com/backstage/community-plugins/tree/main/workspaces/todo/plugins[Todo plugin] directory:
++
+--
+.Example commands to obtain the third-party plugin source code
+[source,bash]
+----
+$ git clone https://github.com/backstage/community-plugins
+$ cd community-plugins/workspaces/todo
+$ yarn install
+----
+--
+
+. *Export backend and front-end plugins*: Run the following commands to build the backend plugin, adjust package dependencies for dynamic loading, and generate self-contained configuration schema:
++
+--
+.Example commands to export a backend plugin
+[source,bash]
+----
+$ cd todo-backend
+$ npx @janus-idp/cli@latest package export-dynamic-plugin
```

Comment:
In the original document, I've included in the examples the output of the commands.
Is there a reason why it can't be output here?

Reply:
I’ve added outputs to the module. To help users distinguish between commands and outputs, I’ve organized them into separate code blocks.